src/Share/Share.router.ts

```diff
@@ -0,0 +1,63 @@
+import { Response } from 'express'
+import { env } from 'decentraland-commons'
+import { format } from 'util'
+
+import { Router } from '../common/Router'
+
+import { Project } from '../Project/Project.model'
+import { withSocialUserAgentDetector, SocialRequest } from '../middleware/share'
+import { Params } from './Share.types'
+import { Pool, PoolAttributes } from '../Pool'
+import { ProjectAttributes } from '../Project'
+import template from './template'
+
+const BUILDER_URL = env.get('BUILDER_URL', '')
+
+export class ShareRouter extends Router {
+  mount() {
+    /**
+     * Redirect to scene
+     */
+    this.router.get(
+      '/share/:type(scene|pool)/:id',
+      withSocialUserAgentDetector,
+      this.redirectToBuilder
+    )
+  }
+
+  private redirectToBuilder = async (req: SocialRequest, res: Response) => {
+    const { type, id } = req.params as Params
+    const targetPath = type === 'scene' ? `/view/${id}` : `/view/${type}/${id}`
+    const url = BUILDER_URL + targetPath
+    if (!req.socialAgent) {
+      return res.redirect(301, url)
+    }
+
+    const p = await this.findPoolOrProject(id, type)
+
+    if (!p) {
+      return res.send(404)
+    }
+
+    const thumbnail =
+      p.thumbnail &&
+      format(
+        `http://localhost:5000/v1/projects/%s/media/thumbnail.png?updated_at=%s`,
```

Comment:
is this correct? shouldn't this be using another URL?

Reply:
Also, what's the advantage of using `util`'s format instead of just using a template string?

```
`http://localhost:5000/v1/projects/${p.id}/media/thumbnail.png?updated_at=${Date.parse(p.updated_at.toString())}\`
```

Reply:
It's the same but looks less wierd than:

<img width="620" alt="Screen Shot 2019-10-29 at 10 33 07" src="https://user-images.githubusercontent.com/208789/67771701-8e5b3c80-fa37-11e9-8ee6-6fdbffd0198c.png">

> ☝️   code after format